.stylelintrc.json

```diff
@@ -0,0 +1,99 @@
+{
+  "extends": "stylelint-config-standard-scss",
```

Comment:
Do we need one of these for CSS too?

Reply:
CSS has a different set of config defaults, but I think this one extends the base one?
When we move away from scss we can change this :)

Reply:
Can you make a quick ticket for us to make one of these for KAIO? So we don't forget, thanks.